app/js/components/manageApps/ManageApps.jsx

```diff
@@ -73,34 +73,52 @@ export default class ManageApps extends React.Component {
   }
 
   handleDrop(files) {
-    if(files.length > 0){
+    if (files.length > 0) {
       this.setState({ files: files });
-    }else{
+    } else {
       this.setState({
         msgBody: "File has not been added, please select a valid zip file",
         msgType: "warning",
         showMsg: true,
       });
-    }    
+    }
   }
 
   handleApplist() {
-    const resultData = [];
+    let installedOwas = [];
```

Comment:
This should be `const` not `let`. const prevents re-assignment not mutation. You are only mutating the array and not re-assigning, hence, const is preferred for this scenario.

Reply:
@tolupatrick004 nice observation.

Reply:
👍